neutone_sdk/audio.py

```diff
@@ -19,8 +23,44 @@
 log = logging.getLogger(__name__)
 
 
+def write_mp3(filename: str, y: np.array, sr: int, quality: float = 0):
```

Comment:
the type here should be `np.ndarray` (`np.array` is the type of the convenience function)

Reply:
Now that I think about it, we never use numpy anywhere in the SDK and always use tensors, so to continue with that trend, why don't we make y a Tensor and do the automatic `.T.numpy()` here and keep the channels and samples check, but swap the axis. 
Most people working with pytorch and audio use `torchaudio.load()` and `torchaudio.save()` since they're typically working with .wav files, so we could keep the `(channels, samples)` dimension order consistency throughout the entire SDK

Reply:
Makes sense, swapped it to that. At the end of the day, they should use the helper function on AudioSample, so maybe it's better to make this private.